README.md

```diff
@@ -3,15 +3,24 @@
 Bundlr validator implementation in Rust
 
 ## Prerequisites
-### Postgresql
-To run a validator locally, you will need to run an instance of Postgresql. The connection string should be contained in the `.env` file. We strongly recommend [running postgres on docker](https://hub.docker.com/_/postgres), 
-### Diesel
-After running Postgresql, install [Diesel](https://diesel.rs/) and run:
+### Arweave Wallet
+Generate an [Arweave wallet](https://docs.arweave.org/info/wallets/arweave-web-extension-wallet), export it to a Json file and save in the project repository (E.g `./wallet.json`)
+
+### Environment Variables
+The following environment variables need to be defined:
+
 ```
-diesel migration run
+DATABASE_URL="./db/validator.db"                                // Path to store database file
+PORT=1234                                                       // The port exposed
+BUNDLER_PUBLIC="OXcT1sVRSA5eGwt2k6Yuz8-3e3g9WJi5uSE99CWqsBs"    // Bundler public key
+VALIDATOR_KEY="./wallet.json"                                   // Path to arweave wallet file
+BUNDLER_URL="https://node1.bundlr.network"                      // Bundler Node url
+```
+
+You can find an example in the `example.env` file. Copy them running:
+```
+cp example.env .env
```

Comment:
Maybe add instructions to fetch the bundler node's JsonWebKey data, save it into a file and use `BUNDLER_KEY` variable pointing to this file instead of the very long string for the public key value.

Reply:
How exactly should that work?

Reply:
The ideal scenario would be to define only `BUNDLER_URL` in the environment and get all the remaining stuff such as public key and gateway.

Reply:
Alright, I think we can leave this for further work. Man-in-the-middle attacks should be mitigated with https fetching.